### PR TITLE
[KFSQA-1173] Fix flickering Account edit ICR accounts AFTs when dynam…

### PR DESCRIPTION
**Pull request by nkimble**
…ic data used for Account contains reference to closed or expired Continuation Account by setting Continuation Account number to new default parameter continuation account IT-CONTINU when business rule error encountered on submit.

### CODE REVIEW

File: features/step_definitions/chart_of_accounts/account.rb [hunk 1]
```diff
@@ -520,6 +520,32 @@
...
+And /^I submit the Account document addressing Continutaion Account errors$/ do
+  # Editting an existing account could cause business rule error that continuation account is closed or expired
+  # upon submit. When those business rule errors occur, edit the continuation account on the account to the default
+  # continutation account parameter value.
+  step 'I submit the Account document'
+
+  #getting errors from page is expensive, obtain reference once that can be reused
+  page_errors = $current_page.errors
+  #search errors array for Continuation Account in any of the error messages
+  continutation_acct_error_exists = false
```
Comment (kyleoliveira) [inline features/step_definitions/chart_of_accounts/account.rb]: `continutation_acct_error_exists` should be `continuation_acct_error_exists`

Fixed typo

Comment (kyleoliveira) [inline features/step_definitions/chart_of_accounts/account.rb]: There's no need to initialize this given the way you set it in the next line. The value will always be overwritten regardless of the outcome from `#any?`

Removed initialization

**Comment by kyleoliveira** [comment]
Should be fine once you've resolved the minor issues here.

**Comment by nkimble** [comment]
All requested changes performed.  PR has been updated with those changes.

**Comment by kyleoliveira** [comment]
Excellent, looks good to go.

**Pull request merged**